dev/CheckBox/CheckBox_themeresources.xaml

```diff
@@ -437,10 +451,20 @@
                                         <ObjectAnimationUsingKeyFrames Storyboard.TargetName="NormalRectangle" Storyboard.TargetProperty="Fill">
                                             <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource CheckBoxCheckBackgroundFillUncheckedPointerOver}" />
                                         </ObjectAnimationUsingKeyFrames>
-                                        <ObjectAnimationUsingKeyFrames Storyboard.TargetName="CheckGlyph" Storyboard.TargetProperty="Foreground">
+                                        <contract7NotPresent:ObjectAnimationUsingKeyFrames Storyboard.TargetName="DownLevelCheckGlyph" Storyboard.TargetProperty="Foreground">
                                             <DiscreteObjectKeyFrame KeyTime="0" Value="{ThemeResource CheckBoxCheckGlyphForegroundUncheckedPointerOver}" />
-                                        </ObjectAnimationUsingKeyFrames>
+                                        </contract7NotPresent:ObjectAnimationUsingKeyFrames>
+                                        <contract7NotPresent:DoubleAnimation Storyboard.TargetName="DownLevelCheckGlyph"
+                                            Storyboard.TargetProperty="Opacity"
+                                            To="0"
+                                            Duration="0" />
+                                        <contract7Present:ObjectAnimationUsingKeyFrames Storyboard.TargetName="CheckGlyph" Storyboard.TargetProperty="Foreground">
```

Comment:
If duration=0 can we just use a Setter instead of animation?

Reply:
I think so, however these specific ones can just be deleted, because 0 is the original value. the other states already had this animation, I think I would prefer to leave it as is and we can make that change seperately

---
In reply to: [580448357](https://github.com/microsoft/microsoft-ui-xaml/pull/4277#discussion_r580448357) [](ancestors = 580448357)